pkg/volume/csi/csi_mounter.go

```diff
@@ -172,28 +228,27 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 	if podAttrs != nil {
-		if attribs == nil {
-			attribs = podAttrs
+		if volAttribs == nil {
```

Comment:
nit: logic here is a little wacky. Can we just do what is in the else block every time? and just initialize volAttribs to not nil somewhere outside of these if statements

Reply:
Will take a look.